Documentation/vmware-backdoor.md

```diff
@@ -24,3 +52,65 @@ are supported by coreos-cloudinit:
 Note: "n", "m", "l", and "x" are 0-indexed, incrementing integers. The
 identifier for the interfaces does not correspond to anything outside of this
 configuration; it is merely for mapping configuration values to each interface.
+
+## GuestInfo Example
+
+Example below provides hostname, DNS address, interface role and static `192.168.178.97` IP address with `255.255.255.0` netmask on ethernet interface which matches `eno1*` interface name and `00:0c:29:63:92:5c` interface mac address (you can simply provide only one match option: `name` or `mac`):
+
+```
+guestinfo.hostname = "coreos"
+guestinfo.interface.0.role = "private"
+guestinfo.dns.server.0 = "8.8.8.8"
+guestinfo.interface.0.route.0.gateway = "192.168.178.1"
+guestinfo.interface.0.route.0.destination = "0.0.0.0/0"
+guestinfo.interface.0.mac = "00:0c:29:63:92:5c"
+guestinfo.interface.0.name = "eno1*"
+guestinfo.interface.0.dhcp = "no"
+guestinfo.interface.0.ip.0.address = "192.168.178.97/24"
+```
+
+CoreOS OVA image contains VMware tools and early-start Cloud-Config which executes `coreos-cloudinit` with `--oem=vmware` option. This option automatically sets `-from-vmware-backdoor` and `-convert-netconf=vmware` flags. Using values above and these flags `coreos-cloudinit` will generate following systemd network unit:
+
+```
+[Match]
+Name=eno1*
+MACAddress=00:0c:29:63:92:5c
+
+[Network]
+DNS=8.8.8.8
+
+[Address]
+Address=192.168.178.97/24
+
+[Route]
+Destination=0.0.0.0/0
+Gateway=192.168.178.1
+```
+
+This unit file will configure matching network interface. If you have set `coreos.config.data` or `coreos.config.url` variables, `coreos-cloudinit` will try to apply these Cloud Configs on VM and substitute `$private_ipv4` and `$public_ipv4` variables if you've configured network interfaces' roles using `interface.<n>.role` guest variable.
+
+## Defining Cloud-Config in GuestInfo
+
+You have to encode your regular Cloud-Config with `base64` tool and set output into `coreos.config.data` variable.
```

Comment:
Oh, interesting. coreos-cloudinit will accept the raw config, but if VIX doesn't support that then I guess we should leave it out of the docs.